dist/windows/installer.nsi

```diff
@@ -41,6 +41,27 @@ Section $(inst_qbt_req) ;"qBittorrent (required)"
   ; Write the installation path into the registry
   WriteRegStr HKLM "Software\qBittorrent" "InstallLocation" "$INSTDIR"
 
+  ; Register qBittorrent as possible default program for .torrent files and magnet links
+  WriteRegStr HKLM "Software\qBittorrent\Capabilities" "ApplicationDescription" "A BitTorrent client in Qt"
```

Comment:
This seems to be machine-wide right? What if the user isn't an admin? What about using `HKCU` instead? `HKLM` can be done later.

Reply:
Currently qBittorrent is installed system-wide and installer requires elevation. So the associated files/protocols are also registered system-wide.
Perhaps you haven't figured out how "Default programs" functions? It is still the will of each user to choose a default program for a particular file type by selecting it from the registered ones.